Formula/tree-sitter.rb

```diff
@@ -0,0 +1,46 @@
+class TreeSitter < Formula
+  desc "Parser generator tool and incremental parsing library"
+  homepage "https://tree-sitter.github.io/"
+  url "https://github.com/tree-sitter/tree-sitter/archive/0.17.3.tar.gz"
+  sha256 "a897e5c9a7ccb74271d9b20d59121d2d2e9de8b896c4d1cfaac0f8104c1ef9f8"
+  head "https://github.com/tree-sitter/tree-sitter.git"
+
+  def install
+    ENV.append_to_cflags "-mmacosx-version-min=10.10"
+    ENV.append "LDFLAGS", "-mmacosx-version-min=10.10"
```

Comment:
Why is this needed?

Reply:
I am honestly not sure—it probably shouldn't be. It was copied from the linked repository. I'll try to remove it and will see if everything still works.